FAQ.md

```diff
@@ -104,10 +104,10 @@ dependency, you have a couple of options:
 2. Use an override.
 
 Overrides are a sledgehammer, and should only be used as a last resort. While
-dependencies and overrides are declared in the same way in `Gopkg.toml`, they
+constraints and overrides are declared in the same way in `Gopkg.toml`, they
 behave differently:
 
-* Dependencies:
+* Constraints:
```

Comment:
Since this is in the context of explaining the diff between `[constraint]` and `[override]`, I agree that it makes more sense to use the term `Constraints`.